FhirToDataLake/deploy/templates/Function/DeployFhirPipelineToFunction.json

```diff
@@ -347,6 +369,10 @@
                             "name": "job__queueUrl",
                             "value": "[concat('https://', variables('storageAccountName'), '.queue.core.windows.net')]"
                         },
+                        {
+                            "name": "job__schedulerCronExpression",
```

Comment:
Shall we change the interval for time-triggered azure function?

Reply:
the time-triggered azure function is only triggered once, and running it always. the interval here is for the scheduler service, I think they are independent.